data/input-contexts/asdw.json

```diff
@@ -1,15 +1,35 @@
 {
-	"context-name": "ASDW",
-	"commands": [
-		{ "command-name": "run", 
-		  "actions": [
-			{ "trigger": "KeyA", "value": -1 },
-			{ "trigger": "KeyD", "value": 1 } 
-		] }, 
-		{ "command-name": "jump", "trigger": "KeyW" },
-		{ "command-name": "down", "trigger": "KeyS" },
-		{ "command-name": "light", "trigger": "KeyR", "event": "Released" },
-		{ "command-name": "heavy", "trigger": "KeyR", "duration": 1000 },
-		{ "command-name": "use", "trigger": "KeyE" }
-	]
+  "context-name": "ASDW_topdown",
```

Comment:
might be good to have the old version for platformer unchanged. Just in case if we make some examples with it later.

Reply:
It was there in the beggining , must have overwritten it by accident ... Will make a fix branch with the old one as well . Thanks !